mpsse! macro has incorrect enums for enable/disable_adaptive_data_clocking

Attempting to use `mpsse!` with the `enable_adaptive_data_clocking`/`disable_adaptive_data_clocking` commands emits code that tries to reference `MpsseCmd::{EnableDataClocking, DisableDataClocking}`, but these enums don't exist. Instead, they're called `EnableAdaptiveClocking` and `DisableAdaptiveClocking`.

Example code (`ft` is of type `&mut libftd2xx::Ft2232h`):

    {
        mpsse! {
            let cmd = {
                disable_adaptive_data_clocking();
                disable_3phase_data_clocking();
            };
        }
        ft.write_all(&cmd).unwrap();
    }

This results in the following compiler error:

    error[E0599]: no variant or associated item named `DisableDataClocking` found for enum `MpsseCmd` in the current scope
      --> src/main.rs:22:9
       |
    22 | /         mpsse! {
    23 | |             let cmd = {
    24 | |                 disable_adaptive_data_clocking();
    25 | |                 disable_3phase_data_clocking();
    26 | |             };
    27 | |         }
       | |         ^
       | |         |
       | |_________variant or associated item not found in `MpsseCmd`
       |           help: there is a variant with a similar name: `Disable3PhaseClocking`
       |
       = note: this error originates in the macro `mpsse` (in Nightly builds, run with -Z macro-backtrace for more info)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,9 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.1.2] - 2024-11-16
 ### Changed
 - Updated the edition from 2018 to 2021.
+
+### Fixed
+- Fixed `enable_adaptive_data_clocking` and `disable_adaptive_data_clocking`
+  functions in the `mpsse!` macro.
 
 ## [0.1.1] - 2022-02-01
 ### Added
@@ -16,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Initial release.
 
-[Unreleased]: https://github.com/ftdi-rs/ftdi-mpsse/compare/v0.1.0...HEAD
+[Unreleased]: https://github.com/ftdi-rs/ftdi-mpsse/compare/v0.1.2...HEAD
+[0.1.2]: https://github.com/ftdi-rs/ftdi-mpsse/compare/v0.1.1...v0.1.2
 [0.1.1]: https://github.com/ftdi-rs/ftdi-mpsse/compare/v0.1.0...v0.1.1
 [0.1.0]: https://github.com/ftdi-rs/ftdi-mpsse/releases/tag/v0.1.0

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ftdi-mpsse"
-version = "0.1.1"
+version = "0.1.2"
 authors = ["Alex Martens <alex@thinglab.org>", "Sergey Matyukevich <geomatsi@gmail.com>"]
 edition = "2021"
 description = "Rust helpers for the MPSSE on FTDI chips."

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1171,10 +1171,10 @@ macro_rules! mpsse {
         mpsse!($passthru {$($tail)*} -> [$($out)* $crate::MpsseCmd::Disable3PhaseClocking as u8,]);
     };
     ($passthru:tt {enable_adaptive_data_clocking(); $($tail:tt)*} -> [$($out:tt)*]) => {
-        mpsse!($passthru {$($tail)*} -> [$($out)* $crate::MpsseCmd::EnableDataClocking as u8,]);
+        mpsse!($passthru {$($tail)*} -> [$($out)* $crate::MpsseCmd::EnableAdaptiveClocking as u8,]);
     };
     ($passthru:tt {disable_adaptive_data_clocking(); $($tail:tt)*} -> [$($out:tt)*]) => {
-        mpsse!($passthru {$($tail)*} -> [$($out)* $crate::MpsseCmd::DisableDataClocking as u8,]);
+        mpsse!($passthru {$($tail)*} -> [$($out)* $crate::MpsseCmd::DisableAdaptiveClocking as u8,]);
     };
     ($passthru:tt {set_gpio_lower($state:expr, $direction:expr); $($tail:tt)*} -> [$($out:tt)*]) => {
         mpsse!($passthru {$($tail)*} -> [$($out)* $crate::MpsseCmd::SetDataBitsLowbyte as u8, $state as u8, $direction as u8,]);

--- a/tests/mpsse-macro.rs
+++ b/tests/mpsse-macro.rs
@@ -179,6 +179,29 @@ fn all_commands_const() {
     assert_eq!(TMS_INDEX, 12);
 }
 
+// https://github.com/ftdi-rs/ftdi-mpsse/issues/17
+#[test]
+fn data_clocking_commands() {
+    mpsse! {
+        const (DATA, DATA_READ_LEN) = {
+            enable_adaptive_data_clocking();
+            disable_adaptive_data_clocking();
+            enable_3phase_data_clocking();
+            disable_3phase_data_clocking();
+        };
+    }
+    assert_eq!(DATA_READ_LEN, 0);
+    assert_eq!(
+        DATA,
+        [
+            MpsseCmd::EnableAdaptiveClocking as u8,
+            MpsseCmd::DisableAdaptiveClocking as u8,
+            MpsseCmd::Enable3PhaseClocking as u8,
+            MpsseCmd::Disable3PhaseClocking as u8,
+        ]
+    );
+}
+
 #[test]
 #[should_panic(expected = "data length must be in 1..=(u16::MAX + 1)")]
 fn clock_data_assert_lower() {

